### DISCUSSION TITLE
Warn when SDL directory doesn't exist

### DISCUSSION THREAD

**Discussion by ghost** [raysan5/raylib#3601] [Ideas]
This is the alternate "fix", still doesn't automatically include the SDL2 from system files, but avoids some confusion (that i had lul)

This is the code, i dont work on makefiles and it uses shell commands, there is a also one that doesnt but looks worse, i dont if this works on windows tho:

```makefile
ifeq ($(PLATFORM),PLATFORM_DESKTOP_SDL)
```
NEW:
```bash
    if [ ! -d "$(SDL_INCLUDE_PATH)" ]; then \
        echo "SDL Include directory doesn't exist: $(SDL_INCLUDE_PATH)"; \
    fi
```
```makefile
    INCLUDE_PATHS += -I$(SDL_INCLUDE_PATH)
endif
```

Here i made a discussion if you have any suggestions, tell me and we can turn this into a PR!

**Comment by raysan5** [comment]
@mr-sihc I think there is no need for that, if the `SDL.h` include is not found, it will complain that it's not found.

**Comment by ghost** [comment]
hmm, well was kinda confusing on why since everything else just used my system install and so when something didnt compile, i am like: "well somethings wrong with the program, everything else i ever compiled with SDL JUST worked!"

maybe someone (not me idk makefiles) can make it use the system dir

OR

we just have SDL as a subproject

or nothing :,(
